include/seqan3/alignment/configuration/align_config_aligned_ends.hpp

```diff
@@ -390,7 +414,25 @@ end_gaps(ends_t && ...) -> end_gaps<std::remove_reference_t<ends_t>...>;
 
 /*!\brief The configuration for aligned sequence ends.
  * \ingroup configuration
- * \tparam end_gaps_t The type of the end gaps. Must be of type seqan3::end_gaps.
+ * \tparam end_gaps_t The type of the end gaps. Must be of type seqan3::align_cfg::end_gaps.
+ *
+ * \details
+ *
+ * This configuration element configures the aligned ends to further refine the global alignment algorithm.
+ * Particularly, the ends of the alignment can be penalised with gap costs or not. For example, the semi-global
+ * alignment does not penalise the leading and trailing gaps of one sequence while it does for the other sequence.
+ *
+ * The class is instantiated with an object of seqan3::align_cfg::end_gaps. The user can configure each of the
+ * gap specifier separately, such that it allows for maximal flexibility to configure the alignment algorithm.
+ * However, there are also predefined seqan3::align_cfg::end_gaps configurations: all_ends_free, none_ends_free,
```

Comment:
I don't know if you want them to link, but if you do, please use `seqan3::align_cfg::all_ends_free` and so on

Reply:
But they won't link for whatever reason. So I removed it. Anyway, I changed it to link to the member group.